src/hub_predtimechart/generate_options.py

```diff
@@ -3,9 +3,33 @@
 
 def ptc_options_for_hub(hub_config: HubConfig):
     """
-    xx
+    Returns predtimechart options dict for `hub_config` - see https://github.com/reichlab/predtimechart?tab=readme-ov-file#options-object .
 
     :param hub_config: a HubConfig
-    :return: xx
     """
-    return {}  # todo xx
+    options = {}
+
+    # todo xx finish!
+
+    # options['target_variables'] = None
+    # options['initial_target_var'] = None
+    # options['task_ids'] = None
+    # options['initial_task_ids'] = None
+
+    options['intervals'] = ["0%", "50%", "95%"]
```

Comment:
Is the `intervals` option related to a hub's output_type_ids for quantile modeling tasks ([for example](https://github.com/cdcepi/FluSight-forecast-hub/blob/main/hub-config/tasks.json#L240))? Or is this different? In either case, how do we determine these three numbers, and will be the same for every hub? 💦

Reply:
Good question. Evan said to hard-code them for now. They are what populates the "Interval" dropdown.